tests/unit/worker/test_submission_worker.py

```diff
@@ -43,7 +162,36 @@ def test_create_dir_as_python_package(self):
 
     def test_return_file_url_per_environment(self):
         returned_url = return_file_url_per_environment(self.url)
-        self.assertEqual(returned_url, "http://testserver/test/url")
+        self.assertEqual(returned_url, "{0}{1}".format(self.testserver, self.url))
+
+    @mock.patch("scripts.workers.submission_worker.create_dir_as_python_package")
+    @mock.patch("scripts.workers.submission_worker.download_and_extract_file")
+    def test_extract_submission_data_success(self, mock_download_and_extract_file, mock_create_dir_as_python_package):
+        patcher_submission_data_dir = mock.patch("scripts.workers.submission_worker.SUBMISSION_DATA_DIR", self.SUBMISSION_DATA_DIR)
+        patcher_submission_input_file_path = mock.patch("scripts.workers.submission_worker.SUBMISSION_INPUT_FILE_PATH", self.SUBMISSION_INPUT_FILE_PATH)
+        patcher_submission_data_dir.start()
+        patcher_submission_input_file_path.start()
+
+        submission = extract_submission_data(self.submission.pk)
+
+        expected_submission_data_dir = self.SUBMISSION_DATA_DIR.format(submission_id=self.submission.pk)
+        mock_create_dir_as_python_package.assert_called_with(expected_submission_data_dir)
+
+        expected_submission_input_file = "{0}{1}".format(self.testserver, self.submission.input_file.url)
+        expected_submission_input_file_path = self.SUBMISSION_INPUT_FILE_PATH.format(submission_id=self.submission.pk, input_file=os.path.basename(self.submission.input_file.name))
```

Comment:
You can use the method to fetch `SUBMISSION_INPUT_FILE_PATH` for a specific submission id. This would be more modular